.travis.yml

```diff
@@ -61,10 +61,36 @@ matrix:
           packages:
             - gcc-multilib
             - libgmp-dev:i386
+    - compiler: gcc
+      env:
+        - BIGNUM=no  ENDOMORPHISM=yes  ASM=x86_64 EXPERIMENTAL=yes ECDH=yes  RECOVERY=yes
```

Comment:
for the record: we *don't have to* disable gmp to get a valgrind pass

Reply:
Yes, But I don't see a lot of value in running valgrind on `gmp` and I'd rather it tested our implementations.
altough there's quite a lot gmp wrapper code too.

Reply:
You might want a second run with endo off.  Otherwise you're not testing the most common config with it, and there have been issues in the past that were valgrind clean with endo on but not off and vice versa.  If speed is a concern, just decrease test count... running tests 4 twice is more testing than tests 8.  If for some reason only one can be run, endo=off is probably better.

Reply:
There is :) https://github.com/bitcoin-core/secp256k1/pull/690/files#diff-354f30a63fb0907d4ad57269548329e3R74